tests/python/gaia-ui-tests/gaiatest/tests/functional/email/test_IMAP_email_notification.py

```diff
@@ -64,6 +64,7 @@ def test_IMAP_email_notification(self):
         self.apps.switch_to_displayed_app()
 
         # check if the sender's email address is fine
+        self.wait_for_condition(lambda m: email.senders_email == mock_email.senders_email)
```

Comment:
This line doesn't exist in [the file in master](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/tests/functional/email/test_IMAP_email_notification.py#L68), `email.wait_for_senders_email_displayed()` is here instead. Why do we need this change here?